autogpt/llm/base.py

```diff
@@ -157,3 +157,13 @@ class ChatModelResponse(LLMResponse):
     """Standard response struct for a response from an LLM model."""
 
     content: str = None
+    function_call: Dict[str, Any] = None
```

Comment:
Please add a proper type for this

![image](https://github.com/Significant-Gravitas/Auto-GPT/assets/12185583/9ee1b59f-065b-41b6-880a-95fe702c66a0)